### ISSUE TITLE
Chrome 48 and Audio

### ISSUE THREAD

**Issue by mabelanski** [muaz-khan/WebRTC-Scalable-Broadcast#13]
Hi

I just download last google Chrome v48
Do you think that bug of audio forward is now fixed ? 

Thanks

**Comment by muaz-khan** [comment]
Yep, this issue (remote-audio processing/relaying) has been fixed since M49.
